Cargo.toml

```diff
@@ -15,6 +15,7 @@ sha1 = "0.6.0"
 
 [dev-dependencies]
 doc-comment = "0.3"
+futures = "0.3"
```

Comment:
I might be overthinking this, but is there any value in making "async/await" a feature so the ```futures``` crate can be excluded if the user doesn't need it?